src/aiomealie/mealie.py

```diff
@@ -142,9 +193,27 @@ async def get_shopping_items(
         params["orderBy"] = ShoppingItemsOrderBy.POSITION
         params["orderDirection"] = OrderDirection.ASCENDING
         params["perPage"] = 9999
-        response = await self._request("api/groups/shopping/items", params)
+        response = await self._get("api/groups/shopping/items", params)
         return ShoppingItemsResponse.from_json(response)
 
+    async def add_shopping_item(
+        self,
+        item: dict[str, Any],
```

Comment:
I think it would be nicer to have this as normal object. you can use `asdict()` to get a dict out of a dataclass

Reply:
Like, or every parameter needs to become a method argument, or we need to use a class. This enforces typesafety checks on the parameters and would decrease the amount of bugs on runtime